lua/resty/moesif/moesif_ser.lua

```diff
@@ -165,11 +165,11 @@ function _M.prepare_message(config)
     user_id_entity = ngx.var.remote_user
   elseif ngx.var.application_id ~= nil then
     user_id_entity = ngx.var.application_id
-  elseif ngx.req.get_headers()[config:get("authorization_header_name")] ~= nil and config:get("authorization_user_id_field") ~= nil then
+  elseif request_headers[string.lower(config:get("authorization_header_name"))] ~= nil and string.lower(config:get("authorization_user_id_field")) ~= nil then
```

Comment:
Are request_headers lowercased already? If not this wont work right?

Reply:
Yes, `request_headers ` are lower cased. Have tested it, works fine.